macaw-aarch32-symbolic/src/Data/Macaw/AArch32/Symbolic/Regs.hs

```diff
@@ -0,0 +1,334 @@
+{-|
+Copyright        : (c) Galois, Inc 2024
+Maintainer       : Langston Barrett <langston@galois.com>
+
+AArch32 registers.
+
+The symbolic execution (and the code discovery in macaw-aarch32) track a
+superset of the user-visible architectural state, which only includes the GPRs
+and SIMD registers.  The extended state includes low-level architectural details
+referenced by the ASL semantics.
+
+In asl-translator, the set of architectural state is referred to as the
+"tracked" registers (or @allGlobalRefs@).  This is state that must be maintained
+during code discovery and symbolic execution, which includes things like:
+
+- The IT state
+- Branch taken/not taken flags
+- Various flags
+
+Note that there are "untracked" state, which is architectural state referred to
```

Comment:
"are untracked state" sounds a bit off grammatically. Perhaps "are pieces of untracked state"?